assets/base/i18n/en.json

```diff
@@ -465,6 +465,11 @@
   "login.signIn": "Log In",
   "login.signingIn": "Logging In",
   "login.username": "Username",
+  "logout.fail.cancel": "Cancel",
+  "logout.fail.confirm": "Confirm",
```

Comment:
Nit, and not strictly my area of expertise, but the language here seems a bit off. The options `Cancel` and `Confirm` don't exactly align with the question `Do you want to continue?` I would expect `No` and `Yes` instead, or to be more explicit `Cancel` and `Continue anyway`. The current answers *do* (kind of) align with the title, though, but there's the concern there that the user only reads the title (`Logout Failed`), skips the long body of text, doesn't understand the implications, and clicks `Confirm`, interpreting it as `Acknowledge`. Maybe the question should be in the title instead?

Also, should there be a `Retry` button?

Reply:
Re: Retry

The user can always hit cancel and retry again, so there is a "way out". But I like the idea of a retry. Nevertheless, too many buttons can be counterproductive. I will leave @abhijit-singh to take the final decision on this one.

Re: Language
@cwarnermm Can you provide your English-fu in this and get us the best copies for these texts? Let me know if you need more context.

Reply:
Here's what I propose:

**Logout not complete**
You’re not fully logged out. Some data might stay accessible on this device until it reconnects to the server. What do you want to do?  

* Try Again
* Log Out Anyway
* Cancel

Where:
* **Try Again** attempts to reconnect to the server and complete the proper logout process to ensure that the user's session is fully terminated both on the server and on the device.
* **Log Out Anyway** prioritizes the user's immediate need to log out of the app locally but leaves some risks associated with lingering data or an active session.

@jupenur @larkox - thoughts?

Reply:
That looks great. Do you have any input on the button order? (just confirming the whether the order you used there is intentional or not)

Also, there is a special case where there is no option to to retry nor cancel. You have already been logged out. This is the current text:

> We could not log you out of the server. Data may continue to be accessible to this device once the device goes back online.

And the button says "Confirm".

Any thoughts on that?

Reply:
Also... I know it says "when the device goes back online" because that is the "most common case", but it could happen also if the server is down, if the network is down, or who knows what that impedes the communication between app and server. 0/5 if we want to be more vague on that part.

Reply:
Note the difference: 

> Some data might stay accessible ***on*** this device ***until*** it reconnects to the server

vs

> data may continue to be accessible ***to*** this device ***once*** the device goes back online

Emphasis mine. The latter is the current text and correct.

I also wonder if the implication of clicking `Log Out Anyway` is clear when the text says `You're not fully logged out`? Would it not be reasonable to assume that clicking `Log Out Anyway` then proceeds to "fully" log me out, instead of leaving the app to the partially logged-out state? Compare with wording like `You cannot be fully logged out`.

Reply:
@larkox - Button order is intentional. I'm expecting users to either try again or log out primarily, but I'm not sure we could definitively know whether one is the more primary action over the other.

For the special use case, I propose updating the text as:

We could not log you out of the server. Some data might stay accessible on this device until it reconnects to the server.  

Let's use **Confirm** as the action if there's precedence elsewhere for a similar enough flow in-product, and **OK** if there isn't.

Reply:
@jupenur also makes important terminology distinctions worth incorporating:

- please update the text in both cases to: "data may continue to be accessible **to** this device **once** the device goes back online", as suggested. No need to bold the product text; emphasis simply to call out changes.
- change **Log Out Anyway** to **Continue Anyway**

Reply:
Thank you @jupenur for ensuring that we're technically accurate with the words we choose to use, and for questioning logic along the way. Very much appreciated!!